src/Tags.zig

```diff
@@ -96,24 +100,101 @@ pub fn findTags(self: *Tags, fname: []const u8) anyerror!void {
     gop.value_ptr.* = {};
 
     const mapped = a: {
-        var file = try std.fs.cwd().openFile(fname, .{});
-        defer file.close();
+        if (OS_TAG == .windows) {
```

Comment:
Let's move all of this into a separate function called `mapFile` or something. Likewise an `unmapFile` for the unmapping stuff. That keeps the `findTags` implementation easier to follow and moves all of the complexity of file mapping into the cross platform `mapFile`/`unmapFile` interface.